Fastfile.private

```diff
@@ -204,36 +229,44 @@ reflected in `Info.plist` during building."
   desc "Keep these new certificates and profiles in sync with Match repository"
   private_lane :create_app do |options|
 
+    build_configuration_key = options[:build_configuration]
+    build_configuration = get_build_configuration(build_configuration: build_configuration_key)
+    app_name = get_application_name(build_configuration: build_configuration_key) % project_name
```

Comment:
I prefer having project_name as a parameter of the function and directly get the ready name.

Reply:
Sorry, I think I didn't explain myself. My idea was instead of having:
`app_name = get_application_name(build_configuration: build_configuration_key) % project_name`
having
`app_name = get_application_name(build_configuration: build_configuration_key, project_name)`

Reply:
Yes, perfect! I'll do that 👍